src/libs/actions/User.ts

```diff
@@ -463,18 +450,17 @@ function isBlockedFromConcierge(blockedFromConciergeNVP) {
     return moment().isBefore(moment(blockedFromConciergeNVP.expiresAt), 'day');
 }
 
-function triggerNotifications(onyxUpdates) {
-    _.each(onyxUpdates, (update) => {
+function triggerNotifications(onyxUpdates: any) {
```

Comment:
I think it should be an array type, but not sure between `any[]` and `unknown[]` 🧐
```suggestion
function triggerNotifications(onyxUpdates: unknown[]) {
```

Reply:
Also, what about `OnyxUpdate`?

Reply:
For me only `any` throws an error, like unexpected any usage.

Reply:
@fabioh8010, any idea why it works differently for us?  

<img width="533" alt="Screenshot 2023-10-03 at 11 51 07" src="https://github.com/Expensify/App/assets/57314631/c64cbf0f-62b0-4933-a632-252f115db003">

Reply:
Hm, works. I just need to use as for a case where types required. No errors, no workarounds.

<img width="629" alt="Screenshot 2023-10-03 at 11 55 09" src="https://github.com/Expensify/App/assets/57314631/030e3ba6-008c-4562-b305-3aec9588bdc4">

Reply:
Well, I see that we are trying to avoid `any` types. You are saying that it is needed here. I still think that it should be an array at least. Let's ask our avocados: @fabioh8010 @gedu.